bin/upgrading/update-db-general.pl

```diff
@@ -18,7 +18,7 @@
 
 mark_clustered(@LJ::USER_TABLES);
 
-register_tablecreate("vgift_ids", <<'EOC');
+register_tablecreate( "vgift_ids", <<'EOC');
```

Comment:
Lolwhut, now it has a space after the open paren but not a space before the close paren